app/controllers/collection_numbers_controller.rb

```diff
@@ -3,39 +3,13 @@
 # Controls viewing and modifying collection numbers.
 class CollectionNumbersController < ApplicationController
   before_action :login_required
-  # except: [
-  #   :index,
-  #   :index_collection_number,
-  #   :list_collection_numbers,
-  #   :collection_number_search,
-  #   :observation_index,
-  #   :show
-  # ]
+
+  # rubocop:disable Rails/LexicallyScopedActionFilter
   before_action :pass_query_params, except: :index
   before_action :store_location, except: [:index, :destroy]
+  # rubocop:enable Rails/LexicallyScopedActionFilter
```

Comment:
```suggestion
  before_action :pass_query_params, only: [
    :show, :new, :create, :edit, :update, :destroy
  ]
  before_action :store_location, only: [
    :show, :new, :create, :edit, :update
  ]
```

Reply:
Is the main reason for this to avoid disabling the cop?  I guess that seems reasonable.  I don't feel strongly about this, but I like that the except list is shorter.